feat(widget): hide orders table

alfetopito opened the pull request:
# Summary

Part of https://github.com/cowprotocol/cowswap/issues/4321
Follow up to https://github.com/cowprotocol/cowswap/pull/4992

Add widget param to hide orders table

![image](https://github.com/user-attachments/assets/6b0200e3-05d3-43da-8745-6287965a7dfa)

# To Test

1. Open widget configurator
2. Check limit and twap forms
* Orders table should be visible
3. Toggle `Hide orders table`
* There shouldn't be a orders table on limit nor twap form
![image](https://github.com/user-attachments/assets/77054eaa-6513-4fd5-be6d-f721a59d6f5a)


* The regular app should work as usual

elena-zh commented:
Hey @alfetopito , it works. 

There is 1 issue: 
1. dApp mode
2. Swap page
3. Hide orders table is ON --> there is no My orders tab on the form. However, it should be displayed
![swap orders](https://github.com/user-attachments/assets/51176807-e603-438c-99c9-c741c4e75fd1)

Also, as we discussed in the meeting. I'd double check with the integrators team about this change as:
1. Users will not see order's status on the UI
2. User will not w able to cancel their open Limit/TWAP orders
3. It won't be possible to find easily an order link on Explorer

alfetopito commented:
> Hey @alfetopito , it works.
> 
> There is 1 issue:
> 
>     1. dApp mode
> 
>     2. Swap page
> 
>     3. Hide orders table is ON --> there is no My orders tab on the form. However, it should be displayed
>        ![swap orders](https://private-user-images.githubusercontent.com/70885163/377006942-51176807-e603-438c-99c9-c741c4e75fd1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjkwODYyMzYsIm5iZiI6MTcyOTA4NTkzNiwicGF0aCI6Ii83MDg4NTE2My8zNzcwMDY5NDItNTExNzY4MDctZTYwMy00MzhjLTk5YzktYzc0MWM0ZTc1ZmQxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDE2VDEzMzg1NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg4MWFkZWI2ZjFkOTQxMjg5YzcwNDhmYzg1ZDlmODIxNDhiZTQzMDlmMWU5YjUxM2I3MzQwODA4ZjlmODAwMGMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.70BUWk0YPNmVsGyDFGRQqHV7G8ngH1AzYDmyNGJHk2o)
> 
> 
> Also, as we discussed in the meeting. I'd double check with the integrators team about this change as:
> 
>     1. Users will not see order's status on the UI
> 
>     2. User will not w able to cancel their open Limit/TWAP orders
> 
>     3. It won't be possible to find easily an order link on Explorer

Added the button back to the SWAP form.

Added a warning in the type documentation.

Will ask Den about it.

elena-zh reviewed (approved):
Thank you!
